### PR TITLE
[17.0][FIX] base_tier_validation: Field merge in view

### PR DESCRIPTION
**Pull request by jaenbox**
Cherry-pick from 16.0 PR #846

@houzefa-abba

### CODE REVIEW

File: base_tier_validation/readme/CONTRIBUTORS.rst
Comment (bosd) [inline base_tier_validation/readme/CONTRIBUTORS.rst:1]: The contributors list has been moved to `CONTRIBUTORS.md` please apply the changes there.

**Review by bosd — CHANGES_REQUESTED** [review]
LGTM,
Minor comments.
In the logs there is a warning message.
Is introduced by this pr?

```

2024-07-16 14:44:05,047 249 WARNING odoo odoo.addons.base.models.ir_model: Two fields (model_name, model_id) of tier.validation.exception() have the same label: Model. [Modules: base_tier_validation and base_tier_validation] 

```

**Comment by jaenbox** [comment]
> Minor comments.
> In the logs there is a warning message.
> Is introduced by this pr?

The warning is not produced by this PR

**Review by houzefa-abba — APPROVED** [review]
Thanks for migrating to 17 :)

LGTM (code review only) aside from the CONTRIBUTORS file that has to be moved

**Comment by LoisRForgeFlow** [comment]
@jaenbox can you attend the issue with the contributors file?

**Comment by jaenbox** [comment]
Sorry, I don't know why I closed the PR after making changes to the CONTRIBUTORS.md. I have created a new one #932 with the cherry-pick and the changes in the CONTRIBUTORS.md. Sorry for the inconvenience.

@LoisRForgeFlow 
@houzefa-abba

**Pull request closed without merging**